### PR TITLE
Support for Symbols in action types

### PR DESCRIPTION
**Pull request by zalmoxisus**
It just reimplements https://github.com/gaearon/redux-devtools-log-monitor/pull/29 and also supports `null` as action type per https://github.com/gaearon/redux-devtools-log-monitor/pull/55.

There's still an error in the demo, coming from `redux-logger`, as it doesn't support symbols.

### CODE REVIEW

File: demo/src/js/DemoApp.jsx [hunk 1]
```diff
@@ -234,7 +234,7 @@ export default connect(
       payload: Array.from({ length: 10000 }).map((_, i) => i)
     }),
     addFunction: () => ({ type: 'ADD_FUNCTION' }),
-    addSymbol: () => ({ type: 'ADD_SYMBOL' }),
+    addSymbol: () => ({ type: Symbol('ADD_SYMBOL') }),
```
Comment (alexkuz) [inline demo/src/js/DemoApp.jsx]: Please add `window.` here - just to make eslint happy

File: demo/src/js/reducers.js [hunk 2]
```diff
@@ -92,6 +92,6 @@ export default {
     ) : state,
   addFunction: (state=null, action) => action.type === 'ADD_FUNCTION' ?
     { f: FUNC } : state,
-  addSymbol: (state=null, action) => action.type === 'ADD_SYMBOL' ?
+  addSymbol: (state=null, action) => action.type === Symbol('ADD_SYMBOL') ?
```
Comment (alexkuz) [inline demo/src/js/reducers.js]: here too

File: src/ActionListRow.jsx [hunk 3]
```diff
@@ -48,7 +48,7 @@ export default class ActionListRow extends Component {
              isInFuture && 'actionListFromFuture'
            ], isSelected, action)}>
         <div {...styling(['actionListItemName', isSkipped && 'actionListItemNameSkipped'])}>
-          {action.type}
+          {action.type !== null && action.type.toString()}
```
Comment (alexkuz) [inline src/ActionListRow.jsx]: I suggest something like this:
```js
action.type === null ?
  '<NULL>' :
  (action.type.toString() || '<EMPTY>')
```

**Review by alexkuz — CHANGES_REQUESTED** [review]

File: src/ActionListRow.jsx (hunk 3 above)
Comment (zalmoxisus) [reply src/ActionListRow.jsx]: That looks great. Will change now. Thanks for the review!

**Comment by zalmoxisus** [comment]
Even though Redux warns when having `undefined` as action type, we should still check it here to avoid unexpected exceptions. For example, when persisting the state, a symbol will be added to localStorage as `undefined`, because `JSON.stringify({ type: Symbol('BECOMES_UNDEFINED') }) === '{}'`. It can be prevented with https://github.com/zalmoxisus/redux-devtools-instrument/pull/14, but the parameter is optional and there could be other cases.

**Pull request merged**